cpp/dolfin/io/XDMFFile.h

```diff
@@ -576,6 +540,9 @@ class XDMFFile : public common::Variable
   // MPI communicator
   dolfin::MPI::Comm _mpi_comm;
 
+  // Integer HDF file identificator
```

Comment:
`// HDF5 file handle`?

Reply:
True handle is the `_hd5_file` pointer. This one is -1 for ASCII encoding or no HDF support. But I'd like to remove this `_h5_id` variable. Its redundant once we keep the `_hdf5_file` also. It introduces now unmanaged depencency - when one changes `_hdf5_file` no change to `_h5_id` is made...

Having just `_hdf5_file` is nice, since it gives access to HDF5File from XDMFFile, but I need to add some interface for it. This will allow together with https://github.com/FEniCS/dolfinx/issues/8 more custom control of XDMFFile.

Reply:
My comment is on the comment. `// HDF5 file handle` would be a better comment.